### PR TITLE
test: migrate Alert, Confirm tests

### PR DESCRIPTION
**Pull request by PavelKirvelTextkernel**
Link [task](https://textkernel.atlassian.net/browse/ONEUI-381)

### CODE REVIEW

File: src/components/Dialogs/Alert/__tests__/Alert.spec.tsx [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-import { mount, ReactWrapper } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
 import { Alert } from '../Alert';
 
 describe('Alert', () => {
     const mockOnAccept = jest.fn();
-    let wrapper: ReactWrapper;
+    let view;
```
Comment (andrentk) [inline src/components/Dialogs/Alert/__tests__/Alert.spec.tsx:9]: type missing

File: src/components/Dialogs/Confirm/__tests__/Confirm.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { Confirm } from '../Confirm';
+
+describe('Confirm', () => {
+    const mockOnAccept = jest.fn();
+    const mockOnCancel = jest.fn();
+    let view;
```
Comment (andrentk) [inline src/components/Dialogs/Confirm/__tests__/Confirm.spec.tsx:10]: type

File: src/components/Dialogs/Alert/__tests__/__snapshots__/Alert.spec.tsx.snap [hunk 3]
```diff
@@ -1,430 +1,5 @@
...
-              </div>
-            </div>
-          </ModalPortal>
-        </Portal>
-      </Modal>
-    </Modal>
-  </Dialog>
-</Alert>
-`;
+exports[`Alert should render correctly 1`] = `<div />`;
```
Comment (eszthoff) [inline src/components/Dialogs/Alert/__tests__/__snapshots__/Alert.spec.tsx.snap:3]: This doesn't look like the DOM we want to render, is it?

File: src/components/Dialogs/Alert/__tests__/__snapshots__/Alert.spec.tsx.snap [hunk 4]
```diff
@@ -1,430 +1,5 @@
...
-              </div>
-            </div>
-          </ModalPortal>
-        </Portal>
-      </Modal>
-    </Modal>
-  </Dialog>
-</Alert>
-`;
+exports[`Alert should render correctly with title 1`] = `<div />`;
```
Comment (eszthoff) [inline src/components/Dialogs/Alert/__tests__/__snapshots__/Alert.spec.tsx.snap:5]: s/a

File: src/components/Dialogs/Alert/__tests__/Alert.spec.tsx (hunk 1 above)
Comment (PavelKirvelTextkernel) [reply src/components/Dialogs/Alert/__tests__/Alert.spec.tsx:9]: :+1:

File: src/components/Dialogs/Confirm/__tests__/Confirm.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        expect(screen.getAllByRole('heading')).toHaveLength(1);
+    });
+
+    it('should call onAccept cb when button is clicked', async () => {
+        const user = userEvent.setup();
+        await user.click(screen.getAllByRole('button')[1]);
+
+        expect(mockOnAccept).toHaveBeenCalledTimes(1);
+    });
+    it('should call onAccept cb when button is clicked', async () => {
```
Comment (eszthoff) [inline src/components/Dialogs/Confirm/__tests__/Confirm.spec.tsx:56]:
```suggestion
    it('should call onCancel callback when button is clicked', async () => {
```

File: src/components/Dialogs/Confirm/__tests__/__snapshots__/Confirm.spec.tsx.snap [hunk 6]
```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Confirm should render correctly 1`] = `<div />`;
```
Comment (eszthoff) [inline src/components/Dialogs/Confirm/__tests__/__snapshots__/Confirm.spec.tsx.snap:3]: s/a

**Review by eszthoff — CHANGES_REQUESTED** [review]

File: src/components/Dialogs/Confirm/__tests__/Confirm.spec.tsx (hunk 2 above)
Comment (PavelKirvelTextkernel) [reply src/components/Dialogs/Confirm/__tests__/Confirm.spec.tsx:10]: :+1:

File: src/components/Dialogs/Confirm/__tests__/__snapshots__/Confirm.spec.tsx.snap (hunk 6 above)
Comment (PavelKirvelTextkernel) [reply src/components/Dialogs/Confirm/__tests__/__snapshots__/Confirm.spec.tsx.snap:3]: :+1:

File: src/components/Dialogs/Alert/__tests__/__snapshots__/Alert.spec.tsx.snap (hunk 4 above)
Comment (PavelKirvelTextkernel) [reply src/components/Dialogs/Alert/__tests__/__snapshots__/Alert.spec.tsx.snap]: :+1:

File: src/components/Dialogs/Alert/__tests__/__snapshots__/Alert.spec.tsx.snap (hunk 3 above)
Comment (PavelKirvelTextkernel) [reply src/components/Dialogs/Alert/__tests__/__snapshots__/Alert.spec.tsx.snap]: :+1:

**Review by eszthoff — APPROVED** [review]

**Pull request merged**